### ISSUE TITLE
Povray materials are not correctly exported

### ISSUE THREAD

**Issue by anhi** [BALL-Project/ball#247]
**Reported by akdehof on 26 Jan 39648677 09:46 UTC**
Pictures generated via the Povray export get far to bright.

**Comment by anhi** [comment]
**Commented by dstoeckel on 10 Mar 39691683 10:40 UTC**
The materials seem to be ok.

However there seems to be a problem with the ambient light intensity. While this seems to be completely ignored by the OpenGL scene, it is written to the POVray input files. Reducing the ambient value to something reasonable like 0.1 gives nice results. (Although, for my taste, we could lower the roughness value to something like 0.2 ...)

So to sum up, this is most likely not a bug in the POVRenderer, but rather in the GLRenderer which ignores the ambient light settings.

**Comment by anhi** [comment]
**Commented by dstoeckel on 15 Mar 39692237 23:33 UTC**
Further narrowing it down: The problem is the MaterialSettingsDialog. Here the preferences for ambient lighting are not stored properly:
  1. Set Ambient to some value other than 0.5
  2. Restart BALLView
  3. The default value of 0.5 has been restored

**Comment by anhi** [comment]
**Commented by dstoeckel on 23 Apr 39694285 12:53 UTC**
Ok, I committed a fix which merely sets the default ambient value to 0.1.
Interestingly I could not reproduce the strange behaviour described in my comment above.
